.github/workflows/check-examples.yml

```diff
@@ -43,6 +43,7 @@ jobs:
     - name: install python requirements for notebooks
       run: |
         python -m pip install --upgrade pip
+        python -m pip install pre-commit==3.0.0
```

Comment:
Is this needed here?

Reply:
Yes, it was in the requirements file, but that is shown on the first page of the manual and I wanted to remove it.

Reply:
![image](https://github.com/CQCL/pytket/assets/70640934/8d81ffc1-3fe1-4cc2-aa1a-141dc51a9b54)